tests/transformers/test_transformers_model_export.py

```diff
@@ -79,23 +101,23 @@ def small_qa_pipeline():
     architecture = "csarron/mobilebert-uncased-squad-v2"
     tokenizer = transformers.AutoTokenizer.from_pretrained(architecture)
     model = transformers.MobileBertForQuestionAnswering.from_pretrained(architecture)
-    return transformers.pipeline(task="question-answering", model=model, tokenizer=tokenizer)
+    yield transformers.pipeline(task="question-answering", model=model, tokenizer=tokenizer)
 
 
 @pytest.fixture(scope="module")
 def small_vision_model():
     architecture = "google/mobilenet_v2_1.0_224"
     feature_extractor = transformers.AutoFeatureExtractor.from_pretrained(architecture)
     model = transformers.MobileNetV2ForImageClassification.from_pretrained(architecture)
-    return transformers.pipeline(
+    yield transformers.pipeline(
```

Comment:
```suggestion
    return transformers.pipeline(
```

We can use `return` if we don't have any teardown code.

Reply:
I had tried putting the cache cleanup call at the conclusion of each fixture after the yield, but that was when using module scoping (where it never got triggered). Switching next commit push to use function-scoping (default) and reverting back to return statements